Content.Server/Arcade/BlockGame/BlockGame.cs

```diff
@@ -83,8 +82,6 @@ private void InvokeGameover()
         {
             _highScorePlacement = _arcadeSystem.RegisterHighScore(meta.EntityName, Points);
             SendHighscoreUpdate();
-            var ev = new MoodEffectEvent("ArcadePlay");
-            _entityManager.EventBus.RaiseLocalEvent(meta.Owner, ev);
```

Comment:
why does this remove the positive moodlet from playing arcades?

Reply:
probably because I'm completely sleep deprived and just copied the ss14 changes.